client/packages/requisitions/src/ResponseRequisition/DetailView/ResponseLineEdit/ResponseEditPage.tsx

```diff
@@ -64,39 +64,35 @@ const ResponseLineEditPageInner = ({
       <DetailContainer>
```

Comment:
I think you can actually remove `DetailContainer` too. I did so in Prescriptions (https://github.com/msupply-foundation/open-msupply/blob/ee9739d4e078218420b4842336434496a381923b/client/packages/invoices/src/Prescriptions/LineEditView/LineEditView.tsx#L151) and I think we benefit from the lesser padding. Any CSS from DetailContainer can be added to `PageLayout` if we really need it.

Then again, can leave that till we do #5973 if you prefer?